lua/telescope/_extensions/neorg/utils.lua

```diff
@@ -0,0 +1,53 @@
+local finders = require "telescope.finders"
+local conf = require("telescope.config").values
+local previewers = require("telescope.previewers")
+
+local ns_previewer = vim.api.nvim_create_namespace "telescope.neorg.previewers"
+
+local utils = {}
+
+utils.copy_table = function(table)
```

Comment:
why is that function needed?

Reply:
This function makes shallow copy of `opts` table that is passed to `find_linkable`.
I use it to be sure that function's  `opts` argument is intact and all changes are applied to it's copy:
https://github.com/nvim-neorg/neorg-telescope/blob/fd62607e3ea22097717c502dd87c5bc8b91dec1a/lua/telescope/_extensions/neorg/find_linkable.lua#L30-L32

Reply:
👍 one small thing though
I really much prefer using `function utils.copy_table(table) ... end` because then lsp goto-definition will work without treating this as two definitions

Reply:
Fixed all function declarations